### PR TITLE
Admin Homepage UI

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Create HomePage UI
## Reference Requirements
- Product Backlog ID: PBI-1
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-anhngo** [comment]
fix conflict

**Comment by at-duyho** [comment]
-please! across review before assign to me.

File: resources/views/backend/homepage/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    <!-- Main content -->
+    <section class="content">
+      <!-- Small boxes (Stat box) -->
+      <div class="row">
+        <div class="col-lg-3 col-xs-6">
+          <!-- small box -->
+          <div class="small-box bg-aqua">
+            <div class="inner">
+              <h3>150</h3>
+
```
Comment (at-anhngo) [inline resources/views/backend/homepage/index.blade.php]: loại xuống không cần thiết

File: resources/views/backend/homepage/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            <a href="#" class="small-box-footer">{{__('home.more_info')}}<i class="fa fa-arrow-circle-right"></i></a>
+          </div>
+        </div>
+        <!-- ./col -->
+        <div class="col-lg-3 col-xs-6">
+          <!-- small box -->
+          <div class="small-box bg-green">
+            <div class="inner">
+              <h3>53<sup style="font-size: 20px">%</sup></h3>
+
```
Comment (at-anhngo) [inline resources/views/backend/homepage/index.blade.php]: xuống dòng không cần thiết

File: resources/views/backend/homepage/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            <a href="#" class="small-box-footer">{{__('home.more_info')}} <i class="fa fa-arrow-circle-right"></i></a>
+          </div>
+        </div>
+        <!-- ./col -->
+        <div class="col-lg-3 col-xs-6">
+          <!-- small box -->
+          <div class="small-box bg-red">
+            <div class="inner">
+              <h3>65</h3>
+
```
Comment (at-anhngo) [inline resources/views/backend/homepage/index.blade.php]: xuống dòng không cần thiết

File: routes/web.php [hunk 4]
```diff
@@ -11,17 +11,14 @@
...
 Auth::routes();
 Route::get('/login', 'Admin\LoginController@showLoginForm')->name('login');
 
 //Admin
 Route::group(['prefix' => 'admin', 'namespace' => 'Admin'], function () {
-	Route::resource('users', 'UserController', ['except' => ['create', 'store']]);
-	Route::resource('books', 'BookController');
+    Route::resource('users', 'UserController', ['except' => ['create', 'store']]);
+    Route::get('/', 'HomeController@index');
+		Route::resource('books', 'BookController');
```
Comment (at-anhngo) [inline routes/web.php]: tấp vào lề

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Review by ghost — APPROVED** [review]

**Comment by at-anhngo** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**